app/views/users/show.html.erb

```diff
@@ -1,3 +1,28 @@
-<h1>
-  this is the detail page for the user with id
-</h1>
\ No newline at end of file
+<a href="<%= user_posts_path(@user) %>">
+  <button id="back-btn">⬅</button>
+</a>
+<ul>
+<li><div class="user-box">
+  <%= render partial: 'pages/user_photo', locals: { user: @user } %>
+  <%= render partial: 'pages/user_details', locals: { user: @user } %>
+</div></li>
+<li><div class="bio">
+  <h3>Bio</h3>
+  <p>
+    <%= @user.bio%>
+  </p>
+</div></li>
+<% @three_recent_posts.each do |post| %>
+    <div class="post bio">
+      <h3>Post <%= raw "#" + post.id.to_s %></h3>
+      <p><%= post.text %></p>
+      <div class="post-intereactions">
+        <span>Comments: <%= post.comments_counter %><br></span>
+        <span>Likes: <%= post.likes_counter %></span>
+      </div>
+   </div>
+<% end %>
+<a href="<%= user_posts_path(@user) %>">
+  <button id="pagination">See all posts</button>
+</a>
+</ul>
```

Comment:
| Your Design | Microverse Design |
|-|-|
| ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/4d1a0a42-dda5-4309-96c9-b2625ef3c45d) | ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/4896b3bb-42bc-4232-ad49-660d531378ee) |

- [ ] It would be nice if you double-check the original design given by microverse and follow that design while making your views to make your UI professional and attractive 👍
- [ ] Kindly follow the below details 👍

1. Kindly make the image on the left side and the size should be similar as you can see in the given design 👍
2. Kindly organize the username and posts_counter on the left as you can see in the design 👀
3. Kindly focus on the comments_counter and likes_counter because they don't show any number 😨